src/parameters/mod.ts

```diff
@@ -124,16 +125,16 @@ export const ParameterVariable = <P extends ParameterDefinition>(
 //   ) as ObjectParameterVariableType<P>;
 // };
 
-// const CreateUntypedObjectParameterVariable = (
-//   namespace: string,
-//   paramName: string,
-// ): UntypedObjectParameterVariableType => {
-//   return WithUntypedObjectProxy(
-//     {},
-//     namespace,
-//     paramName,
-//   ) as UntypedObjectParameterVariableType;
-// };
+export const CreateUntypedObjectParameterVariable = (
```

Comment:
I'm using this as an untyped variable object for `step.outputs` if you call workflow.addStep() with a plain function reference string.